src/controllers/users/getMembersController.ts

```diff
@@ -0,0 +1,30 @@
+import { Response, NextFunction } from 'express';
+import { RequestWithDecoded } from '../../utils';
+import { getMembersQuery } from '../../database';
+
+const getMembersController = async (req: RequestWithDecoded, res: Response, next: NextFunction) => {
+  try {
+    const { page } = req.query;
+
+    const offset = (Number(page || 1) - 1) * 10;
```

Comment:
I think you should multiply the number of pages by 12 not 10 Coz the number of items per page is 12